### ISSUE TITLE
"cabal test" blocks on ssh access to karst.uits.iu.edu

### ISSUE THREAD

**Issue by alxempirical** [hakaru-dev/hakaru#11]
```
Alexs-MacBook-Pro:hakaru alx$ cabal test
Preprocessing test suite 'system-testsuite' for hakaru-0.3.0...
Running 1 test suites...
Test suite system-testsuite: RUNNING...
The authenticity of host 'karst.uits.iu.edu (149.165.234.34)' can't be established.
RSA key fingerprint is SHA256:LFxx+DwrCz79spUAikQ7A3gqTQCn+1MG91+kMtEU400.
Are you sure you want to continue connecting (yes/no)? 
```

**Comment by zaxtax** [comment]
Yep this is known about. Some of the tests require maple to be installed. We are trying to figure out how to disable these tests as a command line option to cabal test.

**Comment by zaxtax** [comment]
Fixed in e2e5379
